app/helpers/results_helper.rb

```diff
@@ -18,8 +18,8 @@ def can_show_remark_request_tab_in_student_pane(assignment, current_user, submis
   end
 
   def student_can_edit_remark_request(submission)
-    !submission.get_remark_result or
-        submission.get_remark_result.marking_state == Result::MARKING_STATES[:unmarked]
+    !submission.remark_result or
+        submission.remark_result.marking_state == Result::MARKING_STATES[:unmarked]
```

Comment:
Line is too long. [83/80]<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.